canonical-websites/maas.io
# Update tour page copy and fix Vanilla 1.8 regressions

## Done

- Updated tour page according to [copydoc](https://docs.google.com/document/d/1KcYwNK0pJyiDVEDq4vTpv6BX_rpSDnxUxtAvAnsRKyw/edit)
- Fixed Vanilla 1.8.0 update regressions

## QA

- Pull code, run `./run`, navigate to http://0.0.0.0:8006/tour
- Check that the copy matches the [copydoc](https://docs.google.com/document/d/1KcYwNK0pJyiDVEDq4vTpv6BX_rpSDnxUxtAvAnsRKyw/edit)
- Check that everything looks good on large and small screens

## Issue / Card

Fixes #267
